### PR TITLE
Explain benefits of registering webhook as an aggregated api

### PR DESCRIPTION
**Pull request by pmorie**
Registering an aggregated API initially stumped me; this PR adds some explanatory text.

### CODE REVIEW

File: artifacts/kube-install/apiserver-list.yaml.template [hunk 1]
```diff
@@ -102,7 +102,15 @@ items:
       # kind is normally the CamelCased singular type. Your resource manifests use this.
       kind: NamespaceReservation
 
-# register as aggregated apiserver
+# register as aggregated apiserver; this has a number of benefits:
```
Comment (tamalsaha) [inline artifacts/kube-install/apiserver-list.yaml.template:105]: May be add in the readme?

Comment (deads2k) [reply artifacts/kube-install/apiserver-list.yaml.template:105]: > May be add in the readme?

Good idea.  He updated the commit to do that too now.

**Comment by deads2k** [comment]
/lgtm

will merge on green.

**Comment by deads2k** [comment]
> @deads2k: changing LGTM is restricted to assignees, and assigning you to the PR failed.

@openshift-ci-robot beware my wrath...

**Pull request merged**